script/spec-runner.js

```diff
@@ -93,6 +100,32 @@ function getSpecHash () {
   ])
 }
 
+function generateHeaders () {
+  return new Promise((resolve, reject) => {
```

Comment:
using a promise here seems like overkill, just `spawnSync` and run away laughing